client/my-sites/people/people-invites/invites-list-end.jsx

```diff
@@ -1,51 +1,45 @@
-import { localize } from 'i18n-calypso';
+import { useTranslate } from 'i18n-calypso';
 import PropTypes from 'prop-types';
-import { PureComponent, Fragment } from 'react';
-import { connect } from 'react-redux';
+import { Fragment, useEffect, useRef } from 'react';
+import { useDispatch } from 'react-redux';
 import ListEnd from 'calypso/components/list-end';
 import { bumpStat } from 'calypso/state/analytics/actions';
 
 import './style.scss';
 
-class InvitesListEnd extends PureComponent {
-	static propTypes = {
-		shown: PropTypes.number,
-		found: PropTypes.number,
-	};
+function InvitesListEnd( { shown, found } ) {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+	const bumped = useRef( false );
 
-	constructor( props ) {
-		super( props );
-		this.bumpedStat = false;
-	}
-
-	// @TODO: Please update https://github.com/Automattic/wp-calypso/issues/58453 if you are refactoring away from UNSAFE_* lifecycle methods!
-	UNSAFE_componentWillReceiveProps( nextProps ) {
-		if ( nextProps.found > nextProps.shown && ! this.bumpedStat ) {
-			this.props.bumpStat( 'calypso_people_invite_list', 'displayed_max' );
-			this.bumpedStat = true;
+	useEffect( () => {
+		if ( found > shown && ! bumped.current ) {
+			dispatch( bumpStat( 'calypso_people_invite_list', 'displayed_max' ) );
+			bumped.current = true;
 		}
-	}
-
-	render() {
-		const { shown, found, translate } = this.props;
+	}, [ dispatch, bumped, found, shown ] );
 
-		return (
-			<Fragment>
-				{ shown < found && (
-					<div className="people-invites__max-items-notice">
-						{ translate(
-							'Showing %(shown)d invite of %(found)d.',
-							'Showing %(shown)d invites of %(found)d.',
-							{ args: { shown, found } }
-						) }
-						<br />
-						{ translate( 'To view more invites, clear some of your existing invites first.' ) }
-					</div>
-				) }
-				<ListEnd />
-			</Fragment>
-		);
-	}
+	return (
+		<Fragment>
+			{ shown < found && (
+				<div className="people-invites__max-items-notice">
+					{ translate(
+						'Showing %(shown)d invite of %(found)d.',
+						'Showing %(shown)d invites of %(found)d.',
+						{ args: { shown, found } }
```

Comment:
I'm seeing the first 100, but below I see a singular count message:

![](https://cldup.com/RF7RIW8FHT.png)

Seems like this needs a `count` arg:

```suggestion
						{
							count: shown,
							args: { shown, found },
						}
```

Reply:
I actually wonder if we need the pluralization at all. We only ever show this paragraph if there's more invites found than we display which means there are >100 invites. Users don't have a way to reduce it to 1 invite per page in which the singular case would be shown 🤔

Reply:
It's already translated, so it doesn't hurt to keep it. But yeah, I agree pluralization is very unnecessary in this case. We definitely won't display 1 per page 😉